bk-nodes/test_bk_nodes.py

```diff
@@ -0,0 +1,66 @@
+import bk_nodes
+import datetime
+import tempfile
+import pytest
+import os
+
+import json
+
+
+# from https://flask.palletsprojects.com/en/1.1.x/testing/
+@pytest.fixture
+def client():
+    app = bk_nodes.app
+    db_fd, app.config['DATABASE'] = tempfile.mkstemp()
+    app.config['TESTING'] = True
+
+    with app.test_client() as client:
+        #with app.app_context():
+        #    init_db()
+        yield client
+
+    os.close(db_fd)
+    os.unlink(app.config['DATABASE'])
+
+
+def test_root(client):
+    rv = client.get('/')
+    assert b'SAGE Beekeeper' in rv.data
+
+
+def test_log_insert(client):
+    bee_db = bk_nodes.BeekeeperDB()
+    bee_db.truncate_table("nodes_log")
+    bee_db.truncate_table("nodes_history")
+
+    test_data = []
+    test_time = datetime.datetime.now().replace(microsecond=0)
+    
+    test_data.append({"node_id": "123", "source": "testing", "operation":"insert", "field_name": "name", "field_value": "Rumpelstilzchen", "effective_time": (test_time -datetime.timedelta(days= 1)).isoformat()})
```

Comment:
Can we insert data for a different node_id ("987") and ensure that is also asserted at the end?

this would ensure that when we insert data for a different node and the `node_history` is re-written that it doesnt mess up other nodes.

Reply:
ok, I just did that, but the change is still local, I forgot to use a different branch for other changes I started